dask/dataframe/groupby.py

```diff
@@ -1784,3 +1784,25 @@ def aggregate(self, arg, split_every=None, split_out=1):
     @derived_from(pd.core.groupby.SeriesGroupBy)
     def agg(self, arg, split_every=None, split_out=1):
         return self.aggregate(arg, split_every=split_every, split_out=split_out)
+
+    #  @derived_from(pd.core.groupby.SeriesGroupBy)
+    def value_counts(self, split_every=None, split_out=1):
+        return self._aca_agg(
+            token="value_counts",
+            func=M.value_counts,
+            aggfunc=lambda s: pd.Series(pd.concat({k: v.sum(level=1) for k, v in s})),
+            split_every=split_every,
+            split_out=split_out,
+        )
+
+    #  @derived_from(pd.core.groupby.SeriesGroupBy)
+    def unique(self, split_every=None, split_out=1):
+        return self._aca_agg(
+            token="unique",
+            func=M.unique,
+            aggfunc=lambda series_gb: pd.Series(
+                {k: v.explode().unique() for k, v in series_gb}
+            ),
```

Comment:
If possible, we prefer to avoid placing dynamicly generated functions into task graphs.  They're more expensive to serialize, which becomes problematic as problem sizes increase.  Instead, we often create small one-line functions at the top level of modules like this.  Here is an example in this module:

https://github.com/dask/dask/blob/8001ee2e258f3f27d40e78990c5b98d44edc138e/dask/dataframe/groupby.py#L532-L533